src/gpuarray/buffer.h

```diff
@@ -634,8 +637,16 @@ GPUARRAY_PUBLIC gpucontext *gpukernel_context(gpukernel *k);
  */
 #define GA_CTX_PROP_MAXLSIZE2 17
 
+/**
+ * Get the vector of collective ops for the context.
+ *
+ * Type: `const gpuarray_comm_ops *`
+ */
+#define GA_CTX_PROP_COMM_OPS  18
```

Comment:
It should be in the master.  I just left the GA_CTX_PROP_BLAS_OP thing there so as not to break existing code for now.